tests/test_databases.py

```diff
@@ -1116,3 +1120,11 @@ async def test_postcompile_queries(database_url):
         results = await database.fetch_all(query=query)
 
         assert len(results) == 0
+
+
+@pytest.mark.parametrize("database_url", DATABASE_URLS)
+@async_adapter
+async def test_supported_backends_schemes(database_url):
```

Comment:
Is this really required? You've added the new url scheme to the github workfllows and that should cover the try/except. Is that correct?

Reply:
I suppose technically not.